documentation/docs/offline/setup.md

```diff
@@ -0,0 +1,46 @@
+---
+id: offlineSetup
+title: Setup
+sidebar_position: 1
+---
+
+The offline library exposes easy to use LiveData/StateFlow objects for messages, reads, typing, members, watchers and more.
```

Comment:
Do we need an article here? Perhaps:
 The offline library exposes **a** easy to use ...

Reply:
I think it's not needed in that case